test/audience.robot

```diff
@@ -27,17 +30,42 @@ Token with correct audience is accepted
     Should Contain   ${out}   404
 
 Token with correct audience in multiple option is accepted
+    [Tags]   audience
     ${uuid}   Generate UUID
     ${se_config}   Get SE config
     ${token}   Get token   scope=-s openid   opts=--aud="${se_config.endpoint} ${uuid}"
     ${url}   SE URL   audience-test-${uuid}
     ${rc}   ${out}   Curl Error   ${url}
     Should Contain   ${out}   404
 
-Token with invalid multiple audiences is rejected
+Token with invalid multiple audiences is rejected (permissive)
+    [Tags]   audience   permissive
+    ${out}   Get token with invalid multiple audience
+    ${ret}   Should Match Regexp   ${out}   40[134]
```

Comment:
```suggestion
    ${ret}   Should Match Regexp   ${out}   40[13]
```
If we first put a file, 404 is not an option.

Reply:
Right. Removed the `PUT` as described in the comment below.